php-timeline.json

```
@@ -64,6 +64,14 @@
         "comment": ""
     },
     {
+        "priority": 2,
+        "date": "2020-01-05T00:00:00+00:00",
+        "title": "Wikipedia moves back to PHP7",
```

PHP 7.0.0 was released on December 3, 2015. That is, at the time when Wikimedia moved to HHVM, PHP 7 was not available yet. 

What do you think about adjusting the wording here?

```suggestion
        "title": "Wikipedia moves back to PHP",
```

---

Changed at 89d30c0!